### ISSUE TITLE
Microsoft closing their APIs!?

### ISSUE THREAD

**Issue by Jeehut** [ayanonagon/Polyglot#24]
こんにちは Ayaka-さん (learning Japanese at the moment 😍 🇯🇵), just wanted to inform you (and potential users of this library) about the fact that Microsoft seems to have decided to close some APIs, including the one used in here. Do you know if they plan to provide machine translation through another service so we can update this library?

Here's the issue on BartyCrouch with the link to the news: https://github.com/Flinesoft/BartyCrouch/issues/54

**Comment by tuzzo77** [comment]
is it true? I have a problem using the Microsoft API

**Comment by pooran** [comment]
Yes they are, but free tier in data market on azure is good enough for medium sized apps

**Comment by Jeehut** [comment]
If anybody is curious, @menzow seems to have found a solution to this problem and is hopefully going to post a PR to this library soon. See her PR on BartyCrouch [here](https://github.com/Flinesoft/BartyCrouch/pull/59).
